nossas/bonde
# Feature community domain settings changes

# Related issues
- Community domain settings changes #674

# How to test
- Log into BONDE
- Choose a community in the community list page
- Click on the **Domínios** left sidebar item
- Check:
  1. It should display a info message on top of the page
  1. The hosted zones table header should be removed

| before                                                                                                                                                                | expected                                                                                                                                                              |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="1266" alt="screen shot 2017-06-20 at 03 53 50" src="https://user-images.githubusercontent.com/5435389/27320356-3f4ef22a-556c-11e7-8960-aed1e99e626f.png"> | <img width="1247" alt="screen shot 2017-06-20 at 03 54 41" src="https://user-images.githubusercontent.com/5435389/27320355-3f4ec642-556c-11e7-918c-0453f1398a05.png"> |

- The DNS check button title text should be standardized with Step 03 of the new domain create

| before                                                                                                                                                                | expected                                                                                                                                                              |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="746" alt="screen shot 2017-06-20 at 03 58 20" src="https://user-images.githubusercontent.com/5435389/27320647-89da9f1e-556d-11e7-8319-221915099166.png">  | <img width="726" alt="screen shot 2017-06-20 at 04 05 26" src="https://user-images.githubusercontent.com/5435389/27320728-d8fb1e52-556d-11e7-9c16-61bfd1505280.png">  |
| <img width="1253" alt="screen shot 2017-06-20 at 03 57 03" src="https://user-images.githubusercontent.com/5435389/27320648-89ded71e-556d-11e7-9b6d-684b324ecf72.png"> | <img width="1245" alt="screen shot 2017-06-20 at 04 05 05" src="https://user-images.githubusercontent.com/5435389/27320729-d8fd346c-556d-11e7-9f86-b37655a94941.png"> |

- It should display the dns check **failure** title text when hover a hosted zone:

<img width="1245" alt="screen shot 2017-06-20 at 04 09 41" src="https://user-images.githubusercontent.com/5435389/27320923-a348e1f8-556e-11e7-9a05-dcc91fddd075.png">

- It should display the dns check **success** title text when hover a hosted zone:

<img width="1248" alt="screen shot 2017-06-20 at 04 10 21" src="https://user-images.githubusercontent.com/5435389/27320944-ba475e3e-556e-11e7-8007-374bc9f5e9e0.png">

- The DNS servers typo should be fixed

| before                                                                                                                                                                | expected                                                                                                                                                              |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="649" alt="screen shot 2017-06-20 at 04 13 12" src="https://user-images.githubusercontent.com/5435389/27321074-4b4277ca-556f-11e7-80ce-6f88f4c9c344.png">  | <img width="667" alt="screen shot 2017-06-20 at 04 15 41" src="https://user-images.githubusercontent.com/5435389/27321073-4b38f4b6-556f-11e7-9828-be3540633820.png">  |

- The DNS check notification text should be changed:

| before                                                                                                                                                                | expected                                                                                                                                                              |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="385" alt="screen shot 2017-06-20 at 04 17 55" src="https://user-images.githubusercontent.com/5435389/27321172-8f7f9012-556f-11e7-9e57-3df90b874444.png">  | <img width="383" alt="screen shot 2017-06-20 at 04 18 10" src="https://user-images.githubusercontent.com/5435389/27321173-8f84d54a-556f-11e7-8522-562a77f92fd9.png">  |

- Check
  1. The **Saiba mais** link on the dns records section should be removed:
  1. The text should be changed

| before                                                                                                                                                                | expected                                                                                                                                                              |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="1150" alt="screen shot 2017-06-20 at 04 19 33" src="https://user-images.githubusercontent.com/5435389/27321231-ca928236-556f-11e7-90d8-8f3563c79dd4.png"> | <img width="1153" alt="screen shot 2017-06-20 at 04 19 39" src="https://user-images.githubusercontent.com/5435389/27321232-ca95315c-556f-11e7-8627-99bc1cf96bdf.png"> |

- The text on the Step 02 of the create domain form should be changed

| before                                                                                                                                                                | expected                                                                                                                                                              |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="1147" alt="screen shot 2017-06-20 at 04 22 33" src="https://user-images.githubusercontent.com/5435389/27321332-412be7c0-5570-11e7-8a8d-5cd25c790f68.png"> | <img width="1147" alt="screen shot 2017-06-20 at 04 23 31" src="https://user-images.githubusercontent.com/5435389/27321331-4124bb30-5570-11e7-97f7-51f58671a9ed.png"> |
